Camera

hillda3141 opened the pull request.

sequoiap commented:
@hillda3141 

Just a few notes here:

* In the examples folder, what is `cameratest.py` for, and is it necessary given there's also `pyro5camera.py`?
* `pyro5camera.py` would probably be better named `uc480.py`. Also, is all of that `set_*()` stuff necessary for the average user? Could those be incorporated into the `__init__()` function, parameterized but with reasonable defaults?
* In both the aforementioned files, there's a lot of commented out lines--if you can remove commented out code that doesn't need to be there, that'd be great.
* One thing we might want to think about in the actual driver code is using a logging module rather than print statements all over the place. Errors should not be printed, they should be raised as custom Exceptions.
* What is the `socketclient.py`, `sockettest.py` files? There are hard coded IP addresses in there--probably undesirable, right?

sequoiap commented:
Also, in `uc480.py`, you should not prepand to `PATH`; the user should do that in their own code before importing PyroLab.

hillda3141 commented:
Thanks for the feedback Sequoia. What I did:
 - Deleted cameratest.py, sockettest.py, and socketclient.py as they were only used to develop small parts of the code and have no real function in the completed project
 - Renamed pyro5camera.py to uc480.py
 - Removed comments and print statements, adding exceptions (I am not sure the best way to do that, so let me know how that looks)
 - Moved the declaration of the os PATH to the server file in pyroservices, but am not sure if that is where you wanted it
 - Made small adjustments to the code while debugging after implementing the changes
